src/view_element.cpp

```diff
@@ -47,6 +47,21 @@
 #include "verse.h"
 #include "vrv.h"
 
+/**
+  * Translates a MEI head.shape attribute value to a SMuFL glph code
+  * (see http://music-encoding.org/documentation/3.0.0/data.HEADSHAPE.list)
+  * Note that only a few of the strings have been mapped thus far.
+ **/
+wchar_t getFontNumber(std::string s)
+{
+    if (s == "filldiamond") return vrv::SMUFL_E0DB_noteheadDiamondBlack;
```

Comment:
Instead of a string s you will then you this->GetHeadShape() which will be a data_HEADSHAPE_list type.  Also, I normally do not vrv:: prefix SMuFL codes.